packages/amplify-category-api/src/provider-utils/awscloudformation/utils/authConfigToAppSyncAuthTypeBiDiMapper.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  AppSyncAuthType,
+  AppSyncAPIKeyAuthType,
+  AppSyncCognitoUserPoolsAuthType,
+  AppSyncOpenIDConnectAuthType,
+} from 'amplify-headless-interface';
+import _ from 'lodash';
+
+export const authConfigToAppSyncAuthType = (authConfig: any = {}): AppSyncAuthType => {
```

Comment:
Could you provide some comment on top these two functions as to what's the purpose of these?

Reply:
done